settings.gradle

```diff
@@ -4,7 +4,7 @@ include 'ndbench-api'
 include 'ndbench-cass-plugins'
 include 'ndbench-cli'
 include 'ndbench-core'
-include 'ndbench-dynamodb-plugins'
+// include 'ndbench-dynamodb-plugins' // Commented as part of https://github.com/Netflix/ndbench/issues/197
```

Comment:
It seems it was fixed as part of #198 Shall we uncomment this back?

Reply:
@ipapapa the DynamoDB plugin is uncommented on the tip of master now. Now, the separate Dax plugin is commented.